app/v3/services/SubmitForeignPropertyBsasService.scala

```diff
@@ -40,33 +40,43 @@ class SubmitForeignPropertyBsasService @Inject()(connector: SubmitForeignPropert
       correlationId: String): Future[Either[ErrorWrapper, ResponseWrapper[Unit]]] = {
 
     val result = for {
-      desResponseWrapper <- EitherT(connector.submitForeignPropertyBsas(request)).leftMap(mapDesErrors(mappingDesToMtdError))
+      desResponseWrapper <- EitherT(connector.submitForeignPropertyBsas(request)).leftMap(mapDesErrors(errorMap))
     } yield desResponseWrapper
```

Comment:
Could shorten this expression to `connector.submitForeignPropertyBsas(request)).map(_.leftMap(mapDesErrors(errorMap))`

Reply:
Not sure if this is necessarily clearer tho, just less code... thoughts?

Reply:
Agree it is simpler.